src/openms/include/OpenMS/DATASTRUCTURES/KDTree.h

```diff
@@ -1787,6 +1788,7 @@ class KDTree : protected _Alloc_base<_Val, _Alloc>
 
 
 
+  // TODO: why not pass node by const-ref?
   std::pair<_Link_type,size_type>
   _M_get_j_max( std::pair<_Link_type,size_type> const node, size_type const level)
```

Comment:
@hannesveit any insights?

Reply:
I don't see why not use &. But should we change the code (since its not ours and it seems cleaner to leave it as is)

Reply:
since this is external code I am a bit hesitant to change it.